### PR TITLE
Create News Page Components

### PR DESCRIPTION
**Pull request by puritybirir**
##### What does this PR do?

- Creates the following News Page components:
   - news top stories 
   - month story
   - News Cards
   - All Stories button
   - News Print Stories
   - News Print Cards
   - Video Audio Stories
- The PR also updates content writeup for any of the available write-ups

##### Description of Task to be completed?
- Create components for the News Page
- Update Content writeup

##### Any background context you want to provide?
- None

##### What are the relevant Pivotal Tracker stories?
#163570086

##### Questions?
- None

##### Screenshots?
screenshot 2019-02-13 at 10 47 43 am
screenshot 2019-02-13 at 10 47 53 am
screenshot 2019-02-13 at 10 48 08 am
screenshot 2019-02-13 at 10 48 17 am

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      [
+        "babel-preset-gatsby",
+        {
+          "targets": {
+            "browsers": [">0.25%", "not dead"],
+          },
+        },
+      ],
+    ],
+    "plugins": ["@babel/plugin-proposal-class-properties"]
```
Comment (kilemensi) [inline .babelrc]: Is there a plugin we're using that requires this? If it's just our code, I'd rather we don't and just stick to ES6

File: src/components/Footer.js [hunk 2]
```diff
@@ -178,10 +182,18 @@ function Footer({ classes }) {
...
+            </a>
+            <a href="https://twitter.com/Code4Africa">
+              <FontAwesomeIcon icon={faFacebookF} color="#ffffff" size="2x" />
+            </a>
+            <a href="https://www.youtube.com/channel/UCa3qX2AwR9WDWO-7_MBL3Hg">
+              <FontAwesomeIcon icon={faYoutube} color="#ffffff" size="2x" />
+            </a>
+            <a href=" https://github.com/CodeForAfrica">
+              <FontAwesomeIcon icon={faGithub} color="#ffffff" size="2x" />
+            </a>
```
Comment (kilemensi) [inline src/components/Footer.js:196]: All links should open on a blank tab with [`rel="noopener noreferrer"`](https://developers.google.com/web/tools/lighthouse/audits/noopener) and for anchors with text, lets use [Link](https://material-ui.com/style/links/) which adheres to Material UI theme settings.

**Review by kilemensi — DISMISSED** [review]
Just small changes @puritybirir.

Buttons still break in Firefox:

screenshot from 2019-02-14 09-51-25

-----------------------------------------

PS: Making this mobile responsive is going to be PITA!

File: .babelrc (hunk 1 above)
Comment (puritybirir) [reply .babelrc]: Done

**Pull request merged**